config/crd/bases/postgresql.cnpg.io_databases.yaml

```diff
@@ -147,7 +147,6 @@ spec:
               template:
                 description: |-
                   The template
-                  TODO: this bit should not be committed as part of this ticket,
```

Comment:
Please @gabriele-wolfox don't delete that. This all is there because the Template should be merged with Nicco's PR, so the TODOs are to help this code stand out.